docs/Project-Configuration/Project-Commands.md

```diff
@@ -597,9 +598,12 @@ tasks:
       - func: ssh-ready
       - func: other-tasks
 ```
-
-The mcipacker.pem key file was created by echoing the value of the
-\${\_\_project_aws_ssh_key_value} expansion into the file. This
+Note:
+- The ${admin_user_name} expansion is set to the value of the
+**user** field set for the command's distro, which can be inspected [on Evergreen's distro page](https://evergreen.mongodb.com/distros).
+- Hosts spawned with `host.create` do not run the distro setup script, so any required initial setup must be done manually.
+- The mcipacker.pem key file was created by echoing the value of the
+\${\_\_project_aws_ssh_key_value} expansion (which gets populated automatically with the ssh public key value) into the file. This
```

Comment:
nit: enclose `${__project_aws_ssh_key_value}` in tildes (also avoids needing to escape underscores).